docs/how-to-guides/themes/theme-json.md

```diff
@@ -1202,6 +1197,14 @@ The default value is 2em.
 }
 ```
 
+The setting for `blockGap` is a boolean, and its default value is `null`. This allows an extra level of control over style output. The `settings.spacing.blockGap` setting in a `theme.json` file accepts the following values:
+
+- `true`: Opt in to displaying _Block spacing_ controls in the editor UI, and output `blockGap` styles.
```

Comment:
```suggestion
- `true`: Opt into displaying _Block spacing_ controls in the editor UI and output `blockGap` styles.
```

Nit: "in to" should generally be combined to "into" when it indicates movement or a change of some kind.

Reply:
Oh, good point. I found myself staring at `opt-in` `opt in` and `opt into` for far too long, thanks for giving an answer! 😄

Reply:
I'm in that same situation most days. Before adding the suggestion I did have to go and ask the grammar gods 😁